src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -382,24 +397,146 @@ export const WorkspaceForm = ({
+        defaultMessage: 'Discard changes',
+      })}
+      buttonColor="danger"
+      defaultFocusedButton="confirm"
+    >
+      {i18n.translate('workspace.form.cancelModal.body', {
+        defaultMessage: 'This will discard all changes. Are you sure?',
+      })}
+    </EuiConfirmModal>
+  );
+
+  // Number of saved changes will be implemented in workspace update page PR
+  const bottomBar = (
+    <div>
+      <EuiSpacer size="xl" />
+      <EuiSpacer size="xl" />
+      <EuiBottomBar>
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s">
+              {opType === WORKSPACE_OP_TYPE_UPDATE && (
+                <EuiText textAlign="left">
+                  {i18n.translate('workspace.form.bottomBar.unsavedChanges', {
+                    defaultMessage: '1 Unsaved change(s)',
+                  })}
+                </EuiText>
+              )}
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="m">
+              <EuiButtonEmpty
+                color="ghost"
+                onClick={showCancelModal}
+                data-test-subj="workspaceForm-bottomBar-cancelButton"
+              >
+                {i18n.translate('workspace.form.bottomBar.cancel', {
+                  defaultMessage: 'Cancel',
+                })}
+              </EuiButtonEmpty>
+              <EuiSpacer />
+              {opType === WORKSPACE_OP_TYPE_CREATE && (
+                <EuiButton
+                  fill
+                  type="submit"
+                  color="primary"
+                  form={formIdRef.current}
+                  data-test-subj="workspaceForm-bottomBar-createButton"
+                >
+                  {i18n.translate('workspace.form.bottomBar.createWorkspace', {
+                    defaultMessage: 'Create workspace',
+                  })}
+                </EuiButton>
+              )}
+              {opType === WORKSPACE_OP_TYPE_UPDATE && (
+                <EuiButton form={formIdRef.current} type="submit" fill color="primary">
+                  {i18n.translate('workspace.form.bottomBar.saveChanges', {
+                    defaultMessage: 'Save changes',
+                  })}
+                </EuiButton>
+              )}
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiBottomBar>
+    </div>
+  );
+
+  const workspaceDetailsTitle = i18n.translate('workspace.form.workspaceDetails.title', {
+    defaultMessage: 'Workspace Details',
+  });
+  const featureVisibilityTitle = i18n.translate('workspace.form.featureVisibility.title', {
+    defaultMessage: 'Feature Visibility',
+  });
+  const usersAndPermissionsTitle = i18n.translate('workspace.form.usersAndPermissions.title', {
+    defaultMessage: 'Users & Permissions',
+  });
+  const libraryCategoryId = i18n.translate('core.ui.libraryNavList.label', {
```

Comment:
Why assign i18n text to a `libraryCategoryId`? Do we need to display `libraryCategoryId`? Perhaps we need to use `DEFAULT_APP_CATEGORIES.opensearchDashboards.id`.